waku/v2/node/wakunode2.nim

```diff
@@ -359,6 +359,20 @@ proc info*(node: WakuNode): WakuInfo =
   let wakuInfo = WakuInfo(listenStr: listenStr)
   return wakuInfo
 
+proc mountLightRelay*(node: WakuNode) = 
```

Comment:
Instead of using "light", what about something like SkeletonRelay? EmptyRelay? NonRelayingRelay? :D

Intuition: with topic sharding a node can subscribe to anything from 0 to 10 topics (say), what exactly is a light node in such a scenario? Add to that that some nodes might do storing, for a certain period of time, for some topics...etc. Desktop node is a good way to think about it, which would be neither "minimally light" nor "maximally full".